src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -78,6 +78,9 @@ static Command parse(String input) throws WrongFormatException, NumberFormatExce
             if(input.indexOf("n/") != input.lastIndexOf("n/")){
                 ui.multipleAdditionErrorMessage();
                 throw new WrongFormatException();
+            } else if(isMultipleInfo(input)){
+                ui.multipleInformationErrorMessage();
+                throw new WrongFormatException();
```

Comment:
Good use of guard clauses, perhaps include a space between the closing parenthesis and opening closing braces